telnyx_common/src/main/java/com/telnyx/webrtc/common/TelnyxViewModel.kt

```diff
@@ -534,6 +558,30 @@ class TelnyxViewModel : ViewModel() {
         }
     }
 
+    fun makePreCallDiagnosis(viewContext: Context, teXMLNumber: String) {
```

Comment:
I would name this something like testDestination or testNumber or something. 

To use it you don't necessarily need to use teXML, some people looking at this to reuse might be confused. 

For that same reason I would actually add documentation above this method.